### PR TITLE
Fix and improve build-from-source instructions.

### PR DESCRIPTION
**Pull request by juliusv**
@fabxc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,9 +44,12 @@ To build Prometheus from the source code yourself you need to have a working
...
 
 You can directly use the `go` tool to download and install the `prometheus`
-and `promtool` binaries into your `GOPATH`:
+and `promtool` binaries into your `GOPATH`. We use Go 1.5's experimental
+vendoring feature, so you will also need to set the `GO15VENDOREXPERIMENT=1`
+environment variable in this case:
 
-    $ go get github.com/prometheus/prometheus/...
+    $ export GO15VENDOREXPERIMENT=1
+    $ go get github.com/prometheus/prometheus/cmd/...
```
Comment (beorn7) [inline README.md]: Why not in one line without polluting the local env?

`GO15VENDOREXPERIMENT=1 go get github.com/prometheus/prometheus/cmd/...`

Comment (juliusv) [inline README.md]: Changed.

**Comment by beorn7** [comment]
:+1:

**Comment by fabxc** [comment]
:+1:

**Pull request merged**